README.md

```diff
@@ -2,13 +2,14 @@
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
+#TABS
+* This is **a project for Software Engineering (SE) students**.<br>
+* The project simulates an ongoing software project for a desktop application designed for *project team leaders* used for assigning tasks and groups.
   * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
   * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
+* It is named TABS because it allows team leaders to keep tabs on their team members.
```

Comment:
Might be a bit awkward. Maybe include the value proposition from UG? Open to suggestions.